index.js

```diff
@@ -149,4 +149,4 @@ function convert(input, shouldLetterBeUpperCase) {
   return input.replace(/./g, (str, i) => (shouldLetterBeUpperCase(str, i) ? str.toUpperCase() : str.toLowerCase()));
 }
 
-module.exports = mOcKiNgCaSe;
+if (typeof module !== 'undefined') module.exports = mOcKiNgCaSe;
```

Comment:
```suggestion
if (typeof module !== 'undefined') {
  module.exports = mOcKiNgCaSe;
}

```

Can you please cover this `if` with tests?

Reply:
I'm not sure how to, as module is a global object unique to the file. I've have a look around.

Reply:
As I think you can redefine  the `module` object inside a `test('', () => {});` and it will be scoped to that test. Try yourself. If you have some issues,  write and I will try to help